projects/js-packages/publicize-components/src/components/share-buttons/share-buttons.tsx

```diff
@@ -59,29 +60,44 @@ export function ShareButtons( { buttonStyle = 'icon', buttonVariant }: ShareButt
 					'text' !== buttonStyle ? <SocialServiceIcon serviceName={ networkName } /> : null;
 
 				return (
-					<Button
-						key={ networkName }
-						icon={ icon }
-						variant={ buttonVariant }
-						aria-label={ label }
-						href={ href }
-						target="_blank"
-						rel="noopener noreferrer"
-						onClick={ getOnClick( href, { network: networkName } ) }
-						data-network={ networkName }
-						className={ styles[ networkName ] }
-					>
-						{ 'icon' !== buttonStyle ? label : null }
-					</Button>
+					<div className={ styles.container } key={ networkName }>
+						<Button
```

Comment:
It would be great if we could make the whole row clickable, rather than just the button. What do you think about adding an :after element to the button that fills the div? So clicking anywhere on the div clicks the button.

There's probably better ways of doing it! I'm CSSing like it's 1999

Reply:
Yes, I think we should make the whole row clickable but that will require quite a bit of changes given the way icon components are designed. It will need icon styling to be separate instead of the button component handling it for us.

Reply:
Done, please check now

Reply:
Yep that works. Do you think we should close the popover when a link it clicked? At the moment we end up with an outline, as the element has focus. Or we could make the focus outline work a bit better

<img width="321" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/437453db-fbfa-4ea8-9568-90e277009d34">

Reply:
I've noticed we haven't got a border on the Copy to clipboard button either. Is that intentional?

Reply:
Fixed